docs/visual-basic/language-reference/operators/nameof.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: "NameOf operator - Visual Basic"
+title: "NameOf operator"
 description: "Learn how to use the NameOf operator in Visual Basic"
 ms.date: 10/27/2019
 helpviewer_keywords:
   - "NameOf operator [Visual Basic]"
 ---
-# NameOf operator - Visual Basic
+# NameOf operator
```

Comment:
@mairaw, I'm not sure if the suffix should be deleted from the h1 or only from the title. Can you confirm ?

Reply:
Also, before I go with further changes, check if these changes are correct. And if there is some cases where I **shouldn't** do them.

Reply:
You can keep them in H1. The suffix is only needed for SEO branding in the title metadata